src/output/gtk-shell.cpp

```diff
@@ -167,8 +167,21 @@ static void handle_gtk_shell_set_startup_id(wl_client *client, wl_resource *reso
  */
 static void handle_gtk_shell_system_bell(wl_client *client, wl_resource *resource,
     wl_resource *surface)
-{}
-
+{
+    view_system_bell_signal data;
+    if (surface)
+    {
+        auto wl_surface =
+            static_cast<wl_resource*>(wl_resource_get_user_data(surface));
+        wayfire_view view = wf::wl_surface_to_wayfire_view(wl_surface);
+
+        if (view)
+        {
+            data.view = view;
+        }
```

Comment:
```suggestion
        data.view = wf::wl_surface_to_wayfire_view(wl_surface);
```

This is equivalent, since data.view is nullptr anyway if view is null.